.travis.yml.mustache

```diff
@@ -35,13 +35,13 @@ language: shell
       set -ex
       sudo chown -R coq:coq /home/coq/${PACKAGE}
       opam install ${PACKAGE} -v -y -j ${NJOBS}
-{{# ci_test_dependants }}
+{{# ci_test_dependants.length }}
```

Comment:
I recall https://github.com/rocq-community/templates/pull/124#pullrequestreview-1853528664 mentioned that `.length` is implementation-specifc. Not sure if it's still a blocking issue.